Failed to build tests on Windows with MSYS2/MinGW-w64

Failed to build target `spdlog-utests` on Windows with MSYS2/MinGW64.

Error below was reported:
```
[ 30%] Building CXX object tests/CMakeFiles/spdlog-utests.dir/test_eventlog.cpp.obj
In file included from X:\path\to\spdlog\tests\test_eventlog.cpp:6:
X:/path/to/spdlog/include/spdlog/sinks/win_eventlog_sink.h: In member function 'void spdlog::sinks::win_eventlog::win_eventlog_sink<Mutex>::sink_it_(const spdlog::details::log_msg&)':
X:/path/to/spdlog/include/spdlog/sinks/win_eventlog_sink.h:222:9: error: 'formatter_' was not declared in this scope; did you mean 'formatter'?
  222 |         formatter_->format(msg, formatted);
      |         ^~~~~~~~~~
      |         formatter
compilation terminated due to -Wfatal-errors.
mingw32-make.exe[3]: *** [tests\CMakeFiles\spdlog-utests.dir\build.make:119: tests/CMakeFiles/spdlog-utests.dir/test_eventlog.cpp.obj] Error 1
mingw32-make.exe[2]: *** [CMakeFiles\Makefile2:170: tests/CMakeFiles/spdlog-utests.dir/all] Error 2
mingw32-make.exe[1]: *** [CMakeFiles\Makefile2:177: tests/CMakeFiles/spdlog-utests.dir/rule] Error 2
mingw32-make.exe: *** [Makefile:222: spdlog-utests] Error 2
```

By adding `this->` before `formatter_`, the error could be fixed:
```C++
this->formatter_->format(msg, formatted);
```

I thought it should be ok to reference a `protected` attribute without `this->`. Maybe another MinGW bug?

--- a/include/spdlog/sinks/win_eventlog_sink.h
+++ b/include/spdlog/sinks/win_eventlog_sink.h
@@ -219,7 +219,7 @@ protected:
         using namespace internal;
 
         memory_buf_t formatted;
-        formatter_->format(msg, formatted);
+        base_sink<Mutex>::formatter_->format(msg, formatted);
         formatted.push_back('\0');
         LPCSTR lp_str = static_cast<LPCSTR>(formatted.data());
 

